Components/grids.css

```diff
@@ -0,0 +1,50 @@
+.grid-50-50{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    max-width: 65rem;
+    grid-auto-rows: minmax(150px, auto);
+    margin: 0 auto;
+    grid-gap: 2rem;
+    
+}
+.grid-wrapper{
+    margin-top: 7rem;
+}
+.one{
+    background: #3bbced;
+    text-align: center;
+    padding: 2rem;
+    color: var(--white-color);
+}
+.two{
+    background: #777;
+    text-align: center;
+    padding: 2rem;
+    color: var(--white-color);
+}
+.grid-com{
+    border: 2px solid var(--light-gray);
+    padding: 1rem;
+    max-width: 70rem;
+}
+.grid-70-30{
+    display: grid;
+    grid-template-columns: 70% 30%;
+    max-width: 65rem;
+    grid-auto-rows: minmax(150px, auto);
+    margin: 0 auto;
+    grid-gap: 2rem;  
+}
+.grid-30-70{
+    display: grid;
+    grid-template-columns: 30% 70%;
+    max-width: 65rem;
+    grid-auto-rows: minmax(150px, auto);
+    margin: 0 auto;
+    grid-gap: 2rem;  
+}
+@media screen and (max-width: 750px) {
```

Comment:
As suggested these width for media screen you can create variable in your root file and then you can use them across the files